numba/cuda/cudadrv/devicearray.py

```diff
@@ -56,6 +56,22 @@ def require_cuda_ndarray(obj):
         raise ValueError('require an cuda ndarray object')
 
 
+def is_contiguous(ary):
```

Comment:
Since this function is new and is only used in `__cuda_array_interface__`, it needs it's own unittests.  If it's behavior should match `array_core(self).flags['C_CONTIGUOUS']`, new tests could be checking against it.